docs/docs/config-reference.md

```diff
@@ -48,6 +48,54 @@ Require that the automerge label (`merge.automerge_label`) be set for Kodiak to
 
 When disabled, Kodiak will immediately attempt to merge any PR that passes all GitHub branch protection requirements.
 
+### `merge.automerge_dependencies.versions`
```

Comment:
Should this setting mention, that the regular rules apply, like a green build on required checks etc?

Reply:
good idea, I've updated to docs to mention that pull requests must pass GitHub branch protections rules to be merged.